packages/@expo/config-types/scripts/generate.js

```diff
@@ -1,28 +1,33 @@
 #!/usr/bin/env node
-import chalk from 'chalk';
-import { Command } from 'commander';
-import fs from 'fs';
-import { compile } from 'json-schema-to-typescript';
-import path from 'path';
-import semver from 'semver';
 
-let version: string = '';
+const chalk = require('chalk');
+const { Command } = require('commander');
+const { compile } = require('json-schema-to-typescript');
+const fs = require('node:fs');
+const path = require('node:path');
+const semver = require('semver');
 
-const packageJSON = require('../package.json');
+let version = '';
+
+const packageJSON = JSON.parse(fs.readFileSync(path.join(__dirname, '../package.json'), 'utf8'));
```

Comment:
What's the benefit of manually reading a JSON file vs importing it?

Reply:
Just not having to deal with `tsconfig.json` changes. Technically, `scripts` isn't included in the `tsconfig.json`'s includes since it alters source output. Other settings could be adjusted for that, and we would also have to add `resolveJsonModules`. But to keep changes to the `tsconfig.json` to a minimum, to avoid risking some accidental breakage I'd overlook, I'm just doing a read file instead